src/fontra/views/editor/scene-model.js

```diff
@@ -716,6 +724,58 @@ export class SceneModel {
   //fontGuidelineSelectionAtPoint(point, size) {
   // }
 
+  backgroundImageSelectionAtPointRect(point = undefined, selRect = undefined) {
```

Comment:
Please make two separate methods, `backgroundImageSelectionAtPoint()` and `backgroundImageSelectionAtRect()`, that then call _this_ function, which should be renamed with an underscore prefix.

Reply:
ok. done.